jobs/cloud_controller_ng/templates/cloud_controller_ng.yml.erb

```diff
@@ -94,6 +94,8 @@ jobs:
   priorities: <%= priorities.to_json %>
   <% end %>
 
+cpu_weight_min_memory: <%= p("cc.cpu_weight_min_memory") %>
```

Comment:
The two new properties need to be added to the other jobs' configuration files, i.e. workers, deployment updater, clock. As the same configuration should be applied everywhere, I would add them to the list of "provided" properties under the `cloud_controller_internal` link:

https://github.com/cloudfoundry/capi-release/blob/cd865562fe6b698f4bd03becb59576fc1359d438/jobs/cloud_controller_ng/spec#L119

Afterwards they can be accessed by `link("cloud_controller_internal").p("cc.cpu_weight_min_memory")`.

Reply:
Now you've added the properties to the individual spec files; this means that when someone wants to change the values, he/she must ensure that this is done consistently for all jobs. I think it would be better not to add them to the spec files, but consume the values through a link as it is done here for example:
https://github.com/cloudfoundry/capi-release/blob/385119fab0eb8a38c5bbbeb05ded70326b198507/jobs/cloud_controller_worker/templates/cloud_controller_ng.yml.erb#L331